xcode/Subconscious/Shared/Components/Deck/DeckView.swift

```diff
@@ -518,20 +518,15 @@ struct DeckModel: ModelProtocol {
                 try? await Task.sleep(for: .seconds(Duration.loading))
                 
                 let us = try await environment.noosphere.identity()
-                let recent = try environment.database.listFeed(owner: us)
+                let recent = try environment.database.listAll(owner: us, limit: 5)
                 let likes = try await environment.userLikes.readOurLikes()
                 
                 var initialDraw = Array(recent
-                    .prefix(10) // take the 10 most recent posts
+                    .prefix(5) // take the 10 most recent posts
```

Comment:
Fix comment

Reply:
Does this really make a large perf difference?

Reply:
No, the main improvement came from reducing the number of entries returned by `listAll`. Before it could `compactMap` up to 1000 rows (which could take hundreds of ms!!?) to only use `10` of them. I deemed that `5` was sufficient for the very limited use we make of this data.